### PR TITLE
community pool liquid stake integration test scripts

### PR DESCRIPTION
**Pull request by sampocs**
## Context
Added a couple small changes while doing manual integration tests. Successfully tested the staking and redemption flow.

## Changelog
* Fixed bug in AccAddress conversion in liquid stake and redeem stake
* Allowed redemptions to ICA accounts
* Created holding addresses in register host
* Removed `community_pool_address` from host zone
* Added stake/redeem/claim scripts
* Added community pool accounts to logs

### CODE REVIEW

File: proto/stride/stakeibc/host_zone.proto [hunk 1]
```diff
@@ -36,8 +36,6 @@ message HostZone {
       [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
   string community_pool_redeem_holding_address = 31
       [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
-  string community_pool_address = 32
-      [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
```
Comment (ethan-stride) [inline proto/stride/stakeibc/host_zone.proto:40]: Good catch, we originally thought this was needed for the call to FundCommunityPool but it is not so we should remove it!

File: dockernet/config/ica_host.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
                     "/cosmos.staking.v1beta1.MsgCancelUnbondingDelegation",
                     "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward",
                     "/cosmos.distribution.v1beta1.MsgSetWithdrawAddress",
+                    "/cosmos.distribution.v1beta1.MsgFundCommunityPool",
```
Comment (ethan-stride) [inline dockernet/config/ica_host.json:20]: This is the whitelist of which msg types are allowed?  Specifically which msg types the host will listen for and in this case the host is DYDX and stride is the client

File: dockernet/scripts/community-pool-staking/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+## Community Pool Staking Integration Tests
+* Set `HOST_CHAINS` to `(DYDX)` in `config.sh`
+* Start dockernet
+```bash
+make start-docker
+```
```
Comment (ethan-stride) [inline dockernet/scripts/community-pool-staking/README.md:6]: call setup.sh to create the connections between dydx and noble?  Or is this not needed for phase1/3

**Review by ethan-stride — COMMENTED** [review]
LGTM.  Was able to run the test locally and it worked smoothly.  Awesome to see it do each step in the balance logs and make it back to the pool

File: dockernet/scripts/community-pool-staking/README.md (hunk 3 above)
Comment (sampocs) [reply dockernet/scripts/community-pool-staking/README.md:6]: not needed for phase 1/3!

Comment (sampocs) [reply dockernet/scripts/community-pool-staking/README.md:6]: but good call that we should update the readme when we go through stage2

**Pull request merged**